Jenkinsfile

```diff
@@ -180,78 +150,63 @@ pipeline {
-                                                }
+                    }
+                }
+                // Coverity will be skipped on all PR builds
+                stage ('Coverity') {
+                    agent { label 'linux-medium' }
+                    steps {
+                        script {
+                            if (env.BRANCH_NAME != 'master') {
+                                echo "Coverity is only run on master"
+                            } else {
+                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                                    withCredentials([string(credentialsId: 'ddf-coverity-token', variable: 'COVERITY_TOKEN')]) {
+                                        withEnv(["PATH=${tool 'coverity-linux'}/bin:${env.PATH}"]) {
+                                            configFileProvider([configFile(fileId: 'coverity-maven-settings', replaceTokens: true, variable: 'MAVEN_SETTINGS')]) {
+                                                echo sh(returnStdout: true, script: 'env')
+                                                sh 'cov-build --dir cov-int mvn -DskipTests=true -DskipStatic=true install -B -pl !$DOCS --settings $MAVEN_SETTINGS'
+                                                sh 'tar czvf ddf.tgz cov-int'
+                                                sh 'curl --form token=$COVERITY_TOKEN --form email=cmp-security-team@connexta.com --form file=@ddf.tgz --form version="master" --form description="Description: DDF CI Build" https://scan.coverity.com/builds?project=codice%2Fddf'
                                             }
                                         }
                                     }
                                 }
                             }
                         }
-                    },
-                    codecov: {
-                        node('linux-large') {
-                            script {
-                                retry(3) {
-                                    checkout scm
-                                }
-                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                                    withCredentials([string(credentialsId: 'DDF_CodeCov', variable: 'DDF_CODECOV_TOKEN')]) {
-                                        sh 'mvn clean install -B -T 1C -pl !$ITESTS'
-                                        sh 'curl -s https://codecov.io/bash | bash -s - -t ${DDF_CODECOV_TOKEN}'
-                                    }
-                                }
+                    }
+                }
+                stage ('Codecov') {
+                    steps {
```

Comment:
I think you might need an `agent { label 'linux-medium' }` definition here

Reply:
Will add :+1: